app/Models/MiscModel.php

```diff
@@ -585,6 +585,23 @@ public function datagridActions(Campaign $campaign): array
                 </a>
             </li>';
             }
+            if ($campaign->enabled('maps') && $this->entityTypeId() == config('entities.ids.map')) {
```

Comment:
This code needs to be on the Map model, since MiscModel doesn't need to know elements specific to Maps